dask/order.py

```diff
@@ -555,7 +570,7 @@ def path_pop() -> Key:
         assert not scrit_path
 
         # A. Build the critical path
-        target = get_target()
+        target = get_target(longest_path=longest_path)
```

Comment:
IIUC this function argument is not be obsolete, isn't it?

Reply:
No, get_target will still be smarter with choosing it's target If one of the hulls is not empty